src/sagemaker/model.py

```diff
@@ -476,9 +476,12 @@ def _script_mode_env_vars(self):
             else:
                 dir_name = self.uploaded_code.s3_prefix
         elif self.entry_point is not None:
-            script_name = self.entry_point
```

Comment:
who uses this script_name

Reply:
This got removed unintentionally, will add it back. It is used elsewhere below in the logic